mavros/src/plugins/local_position.cpp

```diff
@@ -46,7 +47,8 @@ class LocalPositionPlugin : public MavRosPlugin {
 		lp_nh.param<std::string>("tf/frame_id", tf_frame_id, "local_origin");
 		lp_nh.param<std::string>("tf/child_frame_id", tf_child_frame_id, "fcu");
 
-		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local", 10);
+		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local/position", 10);
+		local_velocity = lp_nh.advertise<geometry_msgs::TwistStamped>("local/velocity", 10);
```

Comment:
I see your point about local being redundant.  Though I think that local_position/local is a bad name.  If local_position/**local**/velocity is redundant than local_position/local is really bad naming because it is still obviously local but doesn't tell you what it is.

I would vote to move to
```
local_position/pose
local_position/velocity
```
@vooon does that work?